core/src/main/java/io/questdb/griffin/engine/functions/cast/CastCharToSymbolFunctionFactory.java

```diff
@@ -58,7 +58,7 @@ public Function newInstance(int position, ObjList<Function> args, IntList argPos
         return new Func(arg);
     }
 
-    private static class Func extends AbstractCastToSymbolFunction {
```

Comment:
Unrelated to this change, just something I noticed when reviewing it and maybe worth changing - See the lines 91 and 92: They create 2 different String instances out of the same `StringSink`.
```java
final String str = Chars.toString(sink);
symbols.add(Chars.toString(sink));
return str;
```


That's a waste, just a single instance would be enough.  I can push a change to this branch if that's ok with you.